onnxruntime/python/tools/transformers/large_model_exporter.py

```diff
@@ -0,0 +1,236 @@
+import argparse
+import os
+import shutil
+from pathlib import Path
+
+import onnx
+import torch
+import transformers
+
+
+class Exporter:
+    """
+    A class for exporting large transformer models to ONNX format.
+    """
+
+    def __init__(self):
```

Comment:
The exporter class has a single state model. Would it make sense to move the methods out as functions for readers to reason about them more easily?

Reply:
Thanks. Make sense